data-prepper-plugins/service-map-stateful/build.gradle

```diff
@@ -16,10 +16,13 @@ dependencies {
     compile project(':data-prepper-plugins:common')
     compile project(':data-prepper-plugins:lmdb-prepper-state')
     compile project(':data-prepper-plugins:mapdb-prepper-state')
+    compile 'org.apache.commons:commons-lang3:3.11'
+    testCompile project(':data-prepper-api').sourceSets.test.output
+    implementation "io.micrometer:micrometer-core:1.5.1"
     implementation "com.fasterxml.jackson.core:jackson-databind:${versionMap.jackson_databind}"
     implementation "io.opentelemetry:opentelemetry-proto:${versionMap.opentelemetry_proto}"
     testImplementation "org.hamcrest:hamcrest:${versionMap.hamcrest}"
-    testImplementation "org.mockito:mockito-core:${versionMap.mockito}"
+    testImplementation "org.mockito:mockito-inline:3.6.28"
```

Comment:
any reason for this change here?

Reply:
Keeping it consistent with the Mockito dependency in other plugins. The Otel trace source already has a TODO in the gradle build file to update the version map to a higher version of mockito in all subprojects